SpotifyUserLogin/src/main/java/api/SpotifyInteractor.java

```diff
@@ -203,4 +204,29 @@ public JSONObject getUserProfile(String userId) {
         // Return null after catch (if there is an error)
         return new JSONObject();
     }
+
+    public JSONObject getNonOwnedPlaylists(int limit, int offset) {
```

Comment:
Somewhat curious was the motivation to have this method placed inside the SpotifyInteractor instead of somewhere more closely associated with user data